XCUITests/FxScreenGraph.swift

```diff
@@ -173,6 +173,7 @@ class Action {
     static let SelectTopSitesRows = "SelectTopSitesRows"
 
     static let GoToHomePage = "GoToHomePage"
+    static let GotoHomePage = "BookmarkAll"
```

Comment:
I am not a real reviewer, but isn't here a typo? Should the line be:
```swift
static let BookmarkAll= "BookmarkAll"
```

Reply:
yeah sure